content/en/docs/setup/best-practices/certificates.md

```diff
@@ -57,6 +59,9 @@ Required CAs:
 | front-proxy-ca.crt,key | kubernetes-front-proxy-ca | For the [front-end proxy](/docs/tasks/extend-kubernetes/configure-aggregation-layer/) |
 
 On top of the above CAs, it is also necessary to get a public/private key pair for service account management, `sa.key` and `sa.pub`.
+The diagram below illustrates the CA key and certificate files shown in the previous table that you need to provide to use a single root CA from which all other keys and certificates are generated:
```

Comment:
nit
```suggestion
The diagram below illustrates the CA key and certificate files shown in the previous table:
```
IMO No need to repeat the context given that this paragraph is already Single root CA

Reply:
Done.